lib/utils/compose.coffee

```diff
@@ -421,7 +422,13 @@ pushAndUpdateServiceImages = (docker, token, images, afterEach) ->
 		Promise.map images, ({ serviceImage, localImage, props, logs }, index) ->
 			Promise.join(
 				localImage.inspect().get('Size')
-				progress.push(localImage.name, reporters[index], opts).finally(renderer.end)
+				retry(
+					-> progress.push(localImage.name, reporters[index], opts)
+					3
+					localImage.name
+					2000
+					1.4
```

Comment:
Worth adding a comment beside the value as to what these are for.

Reply:
Done